### ISSUE TITLE
[Partner landing page]: UI issues

### ISSUE THREAD

**Issue by rleitan** [mozilla/voice-web#2053]
[Environment]:
iOS 12.3 -Firefox, Safari
Windows 10- Firefox, Chrome

[STR]:
1.Open https://voice.allizom.org/en/landing/sodedif

[Actual result]:
-Create a profile button instead of "Sign up to create a profile"

-Read more button should have an arrow icon

-A white horizontal line is displayed
![2019-05-28_15h10_58](https://user-images.githubusercontent.com/41474886/58476929-eaedbf00-815a-11e9-8517-809e1cec9643.png)

-mobile only: The Help/Github/Discourse/Contact buttons should be delimited by a grey line: https://app.zeplin.io/project/5a9eec5c53a9a2685450d4be/screen/5ce6e83b2c7c2d1dc8214776

-mobile only: the screenshots and down arrow to expand the section are missing from Benefits

**Comment by Gregoor** [comment]
The copy changed on some elements => https://docs.google.com/document/d/19CEPJY7DD5HrZURx2vJy2YS3SI0jpL4EiAele3O30WI/edit#

Added the arrow icon

Removed the white horizonal line

Mobile footer delimiter: Is that a regression or a new change?

Inlining the screenshots is not sth we currently do on the home page, so to keep the implementation simple, I'd rather not touch it. Is that alright @mbransn?

**Comment by mbransn** [comment]
@Gregoor agreed that the mobile implementation of having slides inline is one that we don't need for this launch. That said, it would be great to capture as a backlog item for all instances of this shared component across the site (homepage and partner landing page). e.g. if you can't build it, log it so we can keep track. :)

**Comment by ftyers** [comment]
Closed as stale.
